core/src/main/scala/kafka/server/ConfigHelper.scala

```diff
@@ -130,7 +131,7 @@ class ConfigHelper(metadataCache: MetadataCache, config: KafkaConfig, configRepo
             else if (resourceNameToBrokerId(resource.resourceName) != config.brokerId)
               throw new InvalidRequestException(s"Unexpected broker id, expected ${config.brokerId} but received ${resource.resourceName}")
             else
-              createResponseConfig(LoggingController.loggers,
+              createResponseConfig(LoggingController.loggers.asScala,
```

Comment:
As logger is not a small collection, could you please consolidate `createResponseConfig` to avoid conversion?

Reply:
Or we can refactor `createResponseConfig` to avoid creating extra collection

Reply:
we can address it in follow-up